packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1573,11 +1636,26 @@ export class ZWaveController
 			);
 		}
 
+		// BUGBUG: fix me
+		// if (options.isLongRange && !this.isLongRange()) {
+		// 	throw new ZWaveError(
+		// 		`Invalid long range inclusion on a non-LR host`,
+		// 		ZWaveErrorCodes.Argument_Invalid,
+		// 	)
+		// }
+		const isLongRange = this.isLongRange();
+
 		// Leave SmartStart listening mode so we can switch to exclusion mode
 		await this.pauseSmartStart();
 
 		this.setInclusionState(InclusionState.Including);
 		this._inclusionOptions = options;
+		// BUGBUG: todo: cache the options used to start inclusion so they can be re-used for stopping, etc
+		this._inclusionFlags = {
+			highPower: true,
+			networkWide: true,
+			protocolLongRange: isLongRange,
+		};
```

Comment:
It seems like you'd want to re-use these same values when issuing the stop commands later? Otherwise you risk starting a long range and stopping a non--long range inclusion?  the spec was unclear on when these flags matter and don't.